### ISSUE TITLE
Missing migrations

### ISSUE THREAD

**Issue by jgoettsch** [divio/cmsplugin-filer#254]
When I ran `migrate` I got a message about missing migrations for cmsplugin-filer. These are the missing migrations:

```
$ python manage.py makemigrations


Migrations for 'cmsplugin_filer_file':
  0005_auto_20160713_1853.py:
    - Alter field cmsplugin_ptr on filerfile
Migrations for 'cmsplugin_filer_image':
  0009_auto_20160713_1853.py:
    - Alter field cmsplugin_ptr on filerimage
Migrations for 'cmsplugin_filer_teaser':
  0003_auto_20160713_1853.py:
    - Alter field cmsplugin_ptr on filerteaser
Migrations for 'cmsplugin_filer_folder':
  0003_auto_20160713_1853.py:
    - Alter field cmsplugin_ptr on filerfolder
Migrations for 'cmsplugin_filer_link':
  0007_auto_20160713_1853.py:
    - Alter field cmsplugin_ptr on filerlinkplugin
Migrations for 'cmsplugin_filer_video':
  0003_auto_20160713_1853.py:
    - Alter field cmsplugin_ptr on filervideo
```

**Comment by sephii** [comment]
FYI django-oscar has a unittest that checks for missing migrations: https://github.com/django-oscar/django-oscar/pull/2063

This could help avoiding future problems.

**Comment by Chronial** [comment]
I actually added a simple version of that test to django-filer recently: https://github.com/divio/django-filer/pull/880/files?diff=unified#diff-02518be55cb68ac739ce9ba80d88c944

**Comment by czpython** [comment]
Fixed by 5937e064ab1d841733b61b7e8bdefe696009f77f

**Comment by czpython** [comment]
Regarding migration tests, this is a great idea!
Thank you, I'll look into this for cmsplugin-filer and the rest of our addons :)
